packages/editor-ui/src/mixins/externalHooks.ts

```diff
@@ -1,16 +1,36 @@
 import type { IExternalHooks } from '@/Interface';
-import type { IDataObject } from 'n8n-workflow';
-import { useWebhooksStore } from '@/stores/webhooks.store';
+import type { PartialDeep } from 'type-fest';
+import type { ExternalHooks, ExternalHooksGenericContext } from '@/types';
 import { defineComponent } from 'vue';
 import { runExternalHook } from '@/utils';
 
+export function extendExternalHooks(hooksExtension: PartialDeep<ExternalHooks>) {
+	if (typeof window.n8nExternalHooks === 'undefined') {
```

Comment:
just to confirm, do we have a way to prevent both hooks scripts running in case the current hooks files also get loaded? or are we waiting on another PR to stop those hooks loading?